fitting/fitScanData.py

```diff
@@ -58,6 +58,8 @@ def fit(self):
             for ch in range(0,128):
                 if self.isDead[vfat][ch]:
                     continue # Don't try to fit dead channels
+		elif not (self.scanHistos[vfat][ch].Integral() > 0):
```

Comment:
Seems to be some whitespace/alignment difference, so I suspect a `TAB` is hanging around
Can you convert all `TAB`s to the appropriate number of `space`s?

Reply:
sigh: done